### DISCUSSION TITLE
Got most of stuff perfect but still need one last thing (thanks for support btw)

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [davatorium/rofi#1451] [Theming] [answered]
So I already got fullscreen working and the colors too

I need one last thing tho. I need there to be five results instead of four. 

Thank you very much.
![2021-09-09-122719_1920x1080_scrot](https://user-images.githubusercontent.com/38611351/132669927-977e5205-7f8a-4efb-9563-e95f14d22db6.png)

**Comment by DaveDavenport** [comment, marked as answer]
Can you post your theme.
If I remember correctly, in fullscreen mode the listview will fill the space allocated to it, so if your margin/padding leaves enough space for 4 rows, it shows 4 rows.
So you need to make sure there is enough space remaining to show 5 lines.
(This method is needed in fullscreen otherwise you get conflicting constraints that won't resolve)

**Reply by SolsticeSpectrum** [reply]
Right. Well is there something like margin-bottom or padding-bottom becasue I don't want to change left, right and top padding since it wouldn't then be like it was before.

**Reply by DaveDavenport** [reply]
Yes, see rofi-theme manpage, it explains this.

**Reply by DaveDavenport** [reply]
```
Format: {Integer}

       • Format: {Distance}

       • Format: {Distance} {Distance}

       • Format: {Distance} {Distance} {Distance}

       • Format: {Distance} {Distance} {Distance} {Distance}

       If no unit is specified, pixels are assumed.

       The different number of fields in the formats are parsed like:

       • 1 field: all

       • 2 fields: topbottom leftright

       • 3 fields: top, leftright, bottom

       • 4 fields: top, right, bottom, left
```

**Reply by SolsticeSpectrum** [reply]
Thanks! One last thing. How do I make that separator line full instead of it having spaces?

**Reply by SolsticeSpectrum** [reply]
It's done https://github.com/DarkReaper231/blacknwhite/blob/master/.config/rofi/config
